libkbfs/disk_block_cache.go

```diff
@@ -32,6 +33,7 @@ import (
 const (
 	// 10 GB maximum storage by default
 	defaultDiskBlockCacheMaxBytes uint64 = 10 * (1 << 30)
+	defaultBlockCacheTableSize    int    = 50 * opt.MiB
```

Comment:
For LevelDB, this is an upper bound rather than fixed size right? Any reason not to make it even bigger?

Reply:
In the average case for our disk cache, it works out to be a fixed size. I'm not sure yet what would be the consequences of making it bigger, other than definitely decreasing the performance efficiency of levelDB (inserts need to sort keys within that 50MiB, and potentially move out elements into the next "level")

Reply:
I see; thanks!